### ISSUE TITLE
hostonly cookie on first visit

### ISSUE THREAD

**Issue by magenx** [nexcess/magento-turpentine#1075]
Varnish 3 and T-0.6.0 creates "httponly" cookie on first visit,
Varnish 4 and latest extension creates "hostonly" + "httponly" cookie, and this breaks login and session. 
How is that possible? how to debug it?

i know there is VCL_FIX to pipe if no frontend cookie, but Varnish 3 and old extension works without any issues. proper cookie created.

**Comment by aricwatson** [comment]
I just tested this using Varnish 4.0.3 and the most recent *devel* branch - I was able to create an account, login/logout/etc without any problems. Can you give me more explicit steps to replicate this bug? The cookies being set appeared to be the same as with Varnish 3.x.

**Comment by magenx** [comment]
without vcl_fix on cached frontend page cookie is set as hostonly, then if you go to login or non-cached page you have another cookie:
```
.www.domain.com
www.domain.com
```

correct cookie with dot ->.www

**Comment by magenx** [comment]
looks like it is something was updated after version 0.6.3
but anyway in latest version i see these two normalization options
```
Normalize Cookie Regex	
Normalized Cookie Target
```

is there any examples on how can i properly use it?

**Comment by aricwatson** [comment]
@magenx Sorry, just re-opened the issue, I was able to replicate the problem in Varnish 4 with the VCL fix disabled. We'll look into a fix.

**Comment by aricwatson** [comment]
@magenx OK, I think I have a fix in PR https://github.com/nexcess/magento-turpentine/pull/1078 - please test if you have a chance! I haven't tested, but I suspect the same issue may affect varnish 3.x.

**Comment by miguelbalparda** [comment]
@magenx did you have the chance to test @aricwatson PR?

**Comment by magenx** [comment]
looks like it works
but we also using regex

**Comment by miguelbalparda** [comment]
This should be fixed by #1078

**Comment by gewaechshaus** [comment]
https://github.com/nexcess/magento-turpentine/issues/1180
